lib/src/components/NxCombobox/types.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as PropTypes from 'prop-types';
+import { omit } from 'ramda';
+import { HTMLAttributes, ReactNode } from 'react';
+import { PublicProps as NxTextInputProps, propTypes as nxTextInputPropTypes } from '../NxTextInput/types';
+
+import DataItem from '../../util/DataItem';
+
+type InputProps = Omit<NxTextInputProps, 'type' | 'value' | 'isPristine' | 'readOnly'>;
+
+export interface Props<T extends string | number = string> extends Omit<HTMLAttributes<HTMLDivElement>, 'onSelect'> {
+  searchText: string;
```

Comment:
Since the idea for `NxComboBox` is to essentially be a text input first and foremost, just with a helpful list, I think its props that implement text input aspects should have the same names that they do on a text input.  That is, instead of `searchText` we should have `value`, and instead of `onSearchTextChange` we should have `onChange`.

Reply:
Updated here d8a0555c5803f8445a8102512197ac9d2f808bb1.